x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -36,18 +57,36 @@ const Details = styled.div`
 
 Details.displayName = 'Details';
 
+const popoverStyle: React.CSSProperties = { position: 'fixed', zIndex: 10 };
```

Comment:
nit - might be able to use `styled` to overwrite `EuiPopover` or inside of `PopoverContainer` add the class name of euiPopover like below

```
const PopoverContainer = styled.div`
  left: -40px;
  position: relative;
  top: 10px;
  .euiPopover {
    position: 'fixed;
   zIndex: 10;
  }
`;
```